core/config/docs/chains-evm.toml

```
@@ -112,6 +112,16 @@ ReaperThreshold = '168h' # Default
 # ResendAfterThreshold controls how long to wait before re-broadcasting a transaction that has not yet been confirmed.
 ResendAfterThreshold = '1m' # Default
 
+[EVM.Transactions.AutoPurge]
+# Enabled enables or disables automatically purging transactions that have been idenitified as terminally stuck (will never be included on-chain). This feature is only expected to be used by ZK chains.
+Enabled = false # Default
+# DetectionApiUrl configures the base url of a custom endpoint used to identify terminally stuck transactions.
+DetectionApiUrl = 'https://venus.scroll.io' # Example
```

Let's not use an actual Scroll API for the example doc.
```suggestion
DetectionApiUrl = 'https://example.api.io' # Example
```

---

Good call! Updated in the new commit